lmdeploy/pytorch/kernels/cuda/w8a8_triton_kernels.py

```diff
@@ -456,17 +551,26 @@ def linear_torch(x, b):
             linear_out_torch.flatten().to(torch.float32)))
 
 
-def test_per_token_quant(x, eps):
+def test_per_token_quant(x, eps, quant_dtype=torch.int8):
```

Comment:
Can we put `test_**` into `tests`?

Reply:
Better not?  These functions are used to test the basic functions(cosine distance to show quant error) and performance for developers

Reply:
Oh, I see. It should be kept